Realm/Tests/QueryTests.m

```diff
@@ -3805,11 +3805,23 @@ - (void)testDictionaryQueryKeySubscriptWithObjectCol {
 - (void)testDictionarySubscriptThrowsException {
     RLMRealm *realm = [self realm];
     RLMAssertThrowsWithReason(([realm objects:@"ArrayPropertyObject" where:@"array['invalid'] = NULL"]),
-                              @"Invalid keypath 'array[\"invalid\"]': only dictionaries support subscript predicates.");
+                              @"Invalid keypath 'array[\"invalid\"]': only dictionaries and mixed support subscript predicates.");
```

Comment:
"mixed" isn't a word that should show up anywhere in our public API, which includes error messages.